src/platform/OpenThread/OpenThreadUtils.cpp

```diff
@@ -95,8 +95,13 @@ void LogOpenThreadStateChange(otInstance * otInst, uint32_t flags)
 {
 #if CHIP_DETAIL_LOGGING
 
+#if OPENTHREAD_API_VERSION >= 126
```

Comment:
is there any value in extracting only the differening item? It would reduce repetidevness of code.

like

```cpp
#if OPENTHREAD_API_VERSION >= 126
   consexpr uint32_t kOtKeyChange = OT_CHANGED_NETWORK_KEY;
#else
   consexpr uint32_t kOtKeyChange = OT_CHANGED_MASTER_KEY;
#endif

constexpr uint32_t kParamsChanged = 
    OT_CHANGED_THREAD_NETWORK_NAME | OT_CHANGED_THREAD_PANID | OT_CHANGED_THREAD_EXT_PANID
    | OT_CHANGED_THREAD_CHANNEL | OT_CHANGED_PSKC | kOtKeyChange;

Reply:
I didn't care much as I expect the old version will be got rid of before CHIP is released, but I can change it.